website_sale_cart_preview/static/src/css/website_sale_cart_preview.css

```diff
@@ -0,0 +1,26 @@
+.mycart-popover .cart_line {
```

Comment:
why didn't you used .less instead of css?, this can break namespaces on general css.